### PR TITLE
docs(setup): add instruction for build toolchain

### PR DESCRIPTION
**Pull request by asudoh**
### Description

Adds some documentation as well as CodeSandbox examples to explain how to use build toolchain.

### Changelog

**New**

- Some documentation as well as CodeSandbox examples to explain how to use build toolchain.

### CODE REVIEW

File: packages/web-components/README.md [hunk 1]
```diff
@@ -87,15 +102,69 @@ Once you do that, you can use our components as easy as using HTML tags, like:
...
-### Expressive Theme
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/web-components/examples/codesandbox/usage/webpack-basic)
+
+> 💡 Above CodeSandbox example uses [`html-webpack-plugin`](https://webpack.js.org/plugins/html-webpack-plugin/) to let [WebPack server](https://webpack.js.org/configuration/dev-server/) serve the `.html` file, but you can use other means to serve `.html` files, for example, using [Express](http://expressjs.com) server.
+
+### Using Sass
+
+While styles are included as part of the web components, setting up Sass toolchain is often useful for styling your contents.
+
+To use Sass, You can add Sass toolchain to your module bundler.
```
Comment (kennylam) [inline packages/web-components/README.md:116]:
```suggestion
To use Sass, you can add Sass toolchain to your module bundler.
```

File: packages/web-components/README.md [hunk 2]
```diff
@@ -67,14 +72,24 @@ For quick start, you can use our pre-built bundle that contains masthead, footer
...
 </html>
 ```
 
-For production usage, our recommendation is **setting up a module bundler** to resolve ECMAScript `import`s. Once you set up a module bundler, you can start importing our component modules, like:
+> 💡 Refer to
+> ["Building for IBM.com'](http://ibmdotcom-web-components.mybluemix.net/?path=/docs/overview-building-for-ibm-dotcom--page) page
+> for `window.digitalData` and `<link rel="alternate" ...>`.
+
+For production usage, our recommendation is **setting up a module bundler** to resolve ECMAScript `import`s.
+You can start with minimum configuration for most module bundlers; For example, with [WebPack](https://webpack.js.org/), you don't need any configuration.
```
Comment (kennylam) [inline packages/web-components/README.md:91]:
```suggestion
You can start with minimum configuration for most module bundlers; For example, with [WebPack](https://webpack.js.org/), you don't need any configuration.
```
```suggestion
You can start with a minimum configuration for most module bundlers. For example, with [WebPack](https://webpack.js.org/), you don't need any configuration.
```

File: packages/web-components/README.md (hunk 1 above)
Comment (kennylam) [inline packages/web-components/README.md:116]:
```suggestion
To use Sass, you can add a Sass toolchain to your module bundler.
```

File: packages/web-components/README.md [hunk 3]
```diff
@@ -87,15 +102,69 @@ Once you do that, you can use our components as easy as using HTML tags, like:
...
+
+### Using Sass
+
+While styles are included as part of the web components, setting up Sass toolchain is often useful for styling your contents.
+
+To use Sass, You can add Sass toolchain to your module bundler.
+A couple of key settings needed in the Sass toolchain are:
+
+1. [`autoprefixer`](https://github.com/postcss/autoprefixer). This is a requirement for using Carbon core Sass code.
+2. `enable-css-custom-properties` Carbon Sass feature flag. This is a requirement for Carbon for IBM.com styles, expecially using the [Expressive theme](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/styles/README.md).
```
Comment (kennylam) [inline packages/web-components/README.md:120]:
```suggestion
2. `enable-css-custom-properties` Carbon Sass feature flag. This is a requirement for Carbon for IBM.com styles, especially using the [Expressive theme](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/styles/README.md).
```

**Review by kennylam — COMMENTED** [review]
Looks great thanks @asudoh! Just a few small typos.

**Comment by asudoh** [comment]
Thank you for spotting those typos @kennylam! Updated.

**Review by kennylam — APPROVED** [review]
Looks good thanks @asudoh!

**Review by jeffchew — APPROVED** [review]

**Pull request merged**